src/Apps/Artist/Routes/AuctionResults/SingleAuctionResultPage/AuctionResult.tsx

```diff
@@ -0,0 +1,102 @@
+import { Box, Column, GridColumns, Join, Spacer, Text } from "@artsy/palette"
+import { AuctionResult_auctionResult$data } from "__generated__/AuctionResult_auctionResult.graphql"
+import { ArtistAuctionResultItemFragmentContainer } from "Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem"
+import { AuctionResultBackLink } from "Apps/Artist/Routes/AuctionResults/SingleAuctionResultPage/AuctionResultBackLink"
+import { AuctionResultImage } from "Apps/Artist/Routes/AuctionResults/SingleAuctionResultPage/AuctionResultImage"
+import { AuctionResultMetaData } from "Apps/Artist/Routes/AuctionResults/SingleAuctionResultPage/AuctionResultMetaData"
+import { AuctionResultPrice } from "Apps/Artist/Routes/AuctionResults/SingleAuctionResultPage/AuctionResultPrice"
+import { AuctionResultTitleInfo } from "Apps/Artist/Routes/AuctionResults/SingleAuctionResultPage/AuctionResultTitleInfo"
+import { MetaTags } from "Components/MetaTags"
+import { createFragmentContainer, graphql } from "react-relay"
+import { extractNodes } from "Utils/extractNodes"
+
+interface AuctionResultProps {
+  auctionResult: AuctionResult_auctionResult$data
+}
+
+export const AuctionResult: React.FC<AuctionResultProps> = ({
+  auctionResult,
+}) => {
+  const { comparableAuctionResults, title, artist } = auctionResult
+
+  const results = extractNodes(comparableAuctionResults)
+
+  return (
+    <>
+      <MetaTags
+        title={`${title}${artist?.name ? ` by ${artist.name}` : ""} | Artsy`}
+      />
+
+      <AuctionResultBackLink auctionResult={auctionResult} />
+
+      <GridColumns mt={[0, 2]} gridRowGap={[2, 0]}>
```

Comment:
Should we be adjusting the grid row gap here? Why not use the default so that it looks uniform to the rest of the site?

Reply:
I'll update it and check with the design team on how it looks :)